js/README.md

```diff
@@ -0,0 +1,76 @@
+# Berty `js/`
+
+**Please, read the main [`README.md`](../README.md) file first.**
+
+This folder contains most of the Typescript and Javascript code needed for the Berty Chat mobile apps.
+
+## Run Berty Chat (macOS)
+
+### 1. Requirements
+
+The installation list may look daunting, but if you have some experience in mobile development, you likely have most of them already!
+
+In addition to the _General requirements for React Native_ below, you'll need:
+
+- An installation of [golang](https://golang.org/dl/)
+- Your `$GOPATH` explicitly set to your golang directory (usually `$HOME/go`)
+- Java 8. If you already have another version of Java, you can use a version manager and Homebrew to add another installation. Some nice instructions are given [here](https://java.christmas/2019/16).
+- The gomobile package: `go get golang.org/x/mobile/cmd/gomobile/... && gomobile init`
+- [Bazel](https://docs.bazel.build/versions/master/install-os-x.html#install-with-installer-mac-os-x)
```

Comment:
is it really mandatory now for mobile dev on Berty?

Reply:
Do you mean having these instructions in general, or a specific item? I wasn't able to run `*.debug` without all of these items.

Reply:
I was asking specifically about bazel

ok, so mandatory :+1:, keep it here